### PR TITLE
feature: SLTP from bookticker. fix: bookTicker typename, depth buffer…

### PR DESCRIPTION
**Pull request by zenixls2**
- add SL/TP through bookTicker
- fix issues in package parsing in binance
- fix issue in error message in depth buffer.
- upgrade ci golang version to 1.18

### CODE REVIEW

File: config/bollmaker.yaml [hunk 1]
```diff
@@ -16,23 +16,23 @@ backtest:
...
-  startTime: "2021-08-01"
-  endTime: "2021-08-30"
+  startTime: "2022-01-01"
+  endTime: "2022-05-12"
   sessions:
   - binance
   symbols:
-  - ETHUSDT
-  account:
+  - MATICBUSD
```
Comment (c9s) [inline config/bollmaker.yaml:24]: Could you copy this config to a new one for bollmaker_matic.yaml? most people trades ETH rather than MATIC. thanks!

**Review by c9s — APPROVED** [review]

**Comment by zenixls2** [comment]
polygon:0x6A384CdD3505bE12A7d4656534742d809FEE0918

**Comment by zenixls2** [comment]
polygon:0x6A384CdD3505bE12A7d4656534742d809FEE0918

**Comment by zenixls2** [comment]
```
polygon:0x6A384CdD3505bE12A7d4656534742d809FEE0918
```

**Comment by zenixls2** [comment]
polygon:0x6A384CdD3505bE12A7d4656534742d809FEE0918

**Pull request merged**